README.md

```diff
@@ -148,6 +148,67 @@ class YouController
 }
 ```
 
+## gRPC
+
+gRPC support was added by the roadrunner-grpc plugin for RoadRunner 2 (https://github.com/spiral/roadrunner-grpc).
+
+To configure Roadrunner for gRPC, refer to the configuration reference at https://roadrunner.dev/docs/beep-beep-grpc.
+
+Basic configuration example:
+
+```
+server:
+    command: "php bin/console baldinof:roadrunner:grpc-worker"
+    # If you are using symfony 5.3+ and the new Runtime component:
+    # remove the previous `command` line above and uncomment the line below.
+    # command: "php public/index.php"
+    env:
+        # APP_RUNTIME: Baldinof\RoadRunnerBundle\Runtime\Runtime
+        APP_RUNTIME: Baldinof\RoadRunnerBundle\Runtime\GrpcRuntime
+
+grpc:
+    listen: "tcp://:9001"
+
+    proto:
+        - "first.proto"
+        - "second.proto"
+```
+
+To register your gRPC services with this bundle, implement the `Baldinof\RoadRunnerBundle\Grpc\GrpcServiceInterface` interface and tag them with `baldinof.roadrunner.grpc_service`:
+
+```
+<?php
+
+declare(strict_types=1);
+
+namespace App\Grpc;
+
+use Baldinof\RoadRunnerBundle\Grpc\GrpcServiceInterface;
+use MyServiceInterface;
+
+final class MyService implements MyServiceInterface, GrpcServiceInterface
```

Comment:
I think we should try to offer an easier way to implement services.

For what I understand [here](https://github.com/spiral/roadrunner-grpc/blob/master/src/Server.php#L77) a gRPC service must implement `Spiral\RoadRunner\GRPC\ServiceInterface`. I think we can do some autoconfiguration:
- if the service implement `ServiceInterface`
- register the service in the server using the domain interface

Example: 
- I have a class `MyDomainService`
- `MyDomainService extends MyDomainServiceInterface`
- `MyDomainServiceInterface extends \Spiral\RoadRunner\GRPC\ServiceInterface`
- we should call `$server->register(MyDomainServiceInterface::class, $myDomainService)`

We should keep a way to configure it manually for edge cases (class implementing multiple interfaces).

What do you think?

Reply:
I see that this can be done more easily and agree with your idea.

In case you have multiple interfaces, what would the difference for Roadrunner be between implementing multiple interfaces on the same Service, or providing it multiple Services with each just one implemented interface?

Reply:
I don't know how the grpc plugin works, but I guess `$server->register()` could be called multiple time with the same instance and different interface as the first argument.

If the class implement several interfaces, it's hard to guess what interface should be used as the first argument of `$server->register()`.

Maybe it can be done by extracting all implemented interfaces that implements `\Spiral\RoadRunner\GRPC\ServiceInterface`?

Reply:
I think I understand what you want to achieve. Can you check my WIP commit https://github.com/Baldinof/roadrunner-bundle/pull/62/commits/264584d72dab56664eecac542802671d2001bc29?
It recursively looks to the class_implements tree of gRPC service interfaces, and lists all interfaces that are NOT `\Spiral\RoadRunner\GRPC\ServiceInterface`.
Did you mean something like this?

Reply:
Yes! Are you ok if I commit to this PR to move this forward?

Reply:
Sure thing!

Reply:
Thanks, I pushed some commits and I validated it's working in a sample project :)

I will rework a little the documentation, and we should be good!